src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterService.java

```diff
@@ -170,9 +170,9 @@ private PGPPublicKey loadPgpPublicKey(String encryptionPublicKey) {
         }
     }
 
-    public LetterStatus getStatus(UUID id, String serviceName) {
+    public LetterStatus getStatus(UUID id) {
         return letterRepository
-            .findByIdAndService(id, serviceName)
```

Comment:
do we still call this from elsewhere? else can delete the underlying code here too I think.

Reply:
Removed

Reply:
Removed